### ISSUE TITLE
try to delete document from collection

### ISSUE THREAD

**Issue by widemar2** [nightroman/Mdbc#22]
I am trying to delete a document in a collection I also looked at this post https://github.com/nightroman/Mdbc/issues/4 but for some reason I can't get the query right.
I attached the examples I was running
[Mongodb_query.txt](https://github.com/nightroman/Mdbc/files/2384538/Mongodb_query.txt)

**Comment by nightroman** [comment]
What is your query in plain English?

**Comment by widemar2** [comment]
All I try is select a distinctive record and delete it.

**Comment by nightroman** [comment]
Your queries probably do not find anything. That is why remove removes nothing.
For example, your

    Remove-MdbcData -Query (New-MdbcQuery Handles -eq '1920') -Result

removes nothing probably because your sample `Handles` is `674`, not '1920' (see your sample).

**Comment by widemar2** [comment]
It is in the collection if you look in the section of the attached doc.
Get-MdbcData

Name                           Value                                                                                                                                                          
----                           -----                                                                                                                                                          
_id                            5b9bfbb78b578b1304153081                                                                                                                                       
Id                             2192                                                                                                                                                           
Name                           DKService                                                                                                                                                      
CPU                            1096.140625                                                                                                                                                    
FileVersion                    19.0.1214.0                                                                                                                                                    
Handles                        674                                                                                                                                                            
_id                            5b9bfbb78b578b1304153082                                                                                                                                       
Id                             1700                                                                                                                                                           
Name                           ccSvcHst                                                                                                                                                       
CPU                            822.859375                                                                                                                                                     
FileVersion                    13.2.1.26                                                                                                                                                      
Handles                        1920

**Comment by nightroman** [comment]
What is the type of the field `Handles`?

**Comment by widemar2** [comment]
It is int32.

**Comment by nightroman** [comment]
> It is int32

But in your query you specify the value as string (mind single quotes).
In other words, remove quotes, pass the number and it should work (unless there is something else).

**Comment by widemar2** [comment]
Thank you so much, that worked I didn't realize I needed to account for the data type in the call.
